app/src/main/java/com/keylesspalace/tusky/service/SendStatusService.kt

```diff
@@ -259,7 +266,41 @@ class SendStatusService : Service(), Injectable {
     private fun cancelSendingIntent(statusId: Int): PendingIntent {
         val intent = Intent(this, SendStatusService::class.java)
         intent.putExtra(KEY_CANCEL, statusId)
-        return PendingIntent.getService(this, statusId, intent, NotificationHelper.pendingIntentFlags(false))
+        return PendingIntent.getService(
+            this,
+            statusId,
+            intent,
+            NotificationHelper.pendingIntentFlags(false)
+        )
+    }
+
+    private fun draftNotification(
```

Comment:
I would rename it to `showDraftNotification()`

Reply:
it is not showing, just building it though